src/utils/quick-open.js

```diff
@@ -112,6 +114,10 @@ export function formatSources(sources: SourcesMap): Array<QuickOpenResult> {
     .filter(source => !isPretty(source))
     .map(source => {
       const sourcePath = getSourcePath(source.url);
+      const sourcePathSplitByPeriod = sourcePath.split(".");
```

Comment:
we might want to have an extension field here which could be a function shared with SourcesTree

Reply:
An extension field in the Source record? or are you talking about a helper function? Seems like the latter.